lang/gcc10-bootstrap/Portfile

```diff
@@ -169,61 +161,124 @@ platform darwin 8 {
                         size    1564560
 
     pre-configure {
+        file mkdir ${workpath}/bins
         system -W ${workpath}/${MAKE_V} "CC=/usr/bin/cc ./configure && make"
         ln -s ${workpath}/${MAKE_V}/make   ${workpath}/bins/make
     }
 
     configure.env-append    MAKE=${workpath}/bins/make
     build.env-append        MAKE=${workpath}/bins/make
-}
 
-set prefix          ${prefix}/libexec/${name}
+    configure.env-replace   PATH=/usr/bin:/bin:/usr/sbin:/sbin \
+                            PATH=${workpath}/bins:/usr/bin:/bin:/usr/sbin:/sbin
+
+    build.env-replace       PATH=/usr/bin:/bin:/usr/sbin:/sbin \
+                            PATH=${workpath}/bins:/usr/bin:/bin:/usr/sbin:/sbin
+}
 
 configure.cmd       ${worksrcpath}/configure
 configure.dir       ${workpath}/build
+configure.pre_args  --prefix=${prefix}/libexec/${name}
 configure.args      --enable-languages=c,c++,objc,obj-c++ \
                     --enable-bootstrap \
                     --with-system-zlib \
+                    --with-local-prefix=${prefix}/libexec/${name} \
```

Comment:
are we really sure about this one? Iain and I have been wondering if this is a good option to use, and the thought has been no. How does it help us here?

Reply:
@kencu it was naive approach to fix `zstd` issue, that I forgot to remove. Let me do it right away.